src/System.Management/src/System.Management.csproj

```diff
@@ -64,14 +64,19 @@
     <Reference Include="System.CodeDom" />
     <Reference Include="System.Security.Permissions" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'uap'">
-    <Reference Include="System" />
+  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
```

Comment:
This `ItemGroup` was added specifically for uap, we should be able to remove it no?

Reply:
Well it is needed for netcoreapp and netcoreapp2.0 as well. That is why I added more references and deleted the reference to System which is not needed anymore.

Reply:
Ah I didn't notice the additions below.